src/main/groovy/com/marklogic/gradle/task/MlcpTask.groovy

```diff
@@ -177,17 +184,21 @@ class MlcpTask extends JavaExec {
 		// Ensure connection arguments are present, but not if a COPY
 		boolean isCopy = "COPY".equals(command)
 		if (!isCopy) {
-			if (!newArgs.contains("-host")) {
-				newArgs.add("-host")
-				newArgs.add(config.getHost())
+			if (!optionsFileFields.contains("host")) {
```

Comment:
Can combine these into a single if

Reply:
Also thinking that the code would be nicer if the host/username/password stuff were in a new method called "addArgsFromAppConfig". That would make the intent more clear and also shorten up this method.

Reply:
That's fair.

Reply:
I went ahead and changed it. So, I'll ask for a re-review.